tests/foreman/api/test_errata.py

```diff
@@ -913,3 +920,141 @@ def test_errata_installation_with_swidtags(
+    )
+    # Fetch available subscriptions
+    subs = entities.Subscription(organization=module_manifest_org).search(
+        query={'search': f'{DEFAULT_SUBSCRIPTION_NAME}'}
+    )
+    assert subs
+    # Add default subscription to activation key
+    rhel8_module_ak.add_subscriptions(data={'subscription_id': subs[0].id})
+    # Add custom subscription to activation key
+    product = entities.Product(organization=module_manifest_org).search(
+        query={'search': "redhat=false"}
+    )
+    custom_sub = entities.Subscription(organization=module_manifest_org).search(
+        query={'search': f"name={product[0].name}"}
+    )
+    rhel8_module_ak.add_subscriptions(data={'subscription_id': custom_sub[0].id})
+    return rhel8_module_ak
+
+
+@pytest.mark.tier2
+def test_apply_modular_errata_using_default_content_view(
+    module_manifest_org, default_lce, rhel8_contenthost, rhel8_module_ak, rhel8_custom_repo_cv
+):
+    """
+    Registering a RHEL8 system to the default content view with no modules enabled results in
+    no modular errata or packages showing as applicable or installable
+
+    Enabling a module on a RHEL8 system assigned to the default content view and installing an
+    older package should result in the modular errata and package showing as applicable and
+    installable
+
+
+    :id: 030981dd-19ba-4f8b-9c24-0aee90aaa4c4
+
+
+    Steps:
+        1. Register host with AK, install tools
+        2. Assert no errata indicated
+        3. Install older version of stream
+        4. Assert errata is applicable
+        5. Update module stream
+        6. Assert errata is no longer applicable
+
+    :expectedresults:  Errata enumeration works with module streams when using default Content View
+
+    :CaseAutomation: Automated
+
+    :CaseImportance: Critical
+
+    :CaseLevel: System
+    """
+    module_name = 'duck'
+    stream = '0'
+    version = '20180704244205'
+    entities.Parameter(
+        name='remote_execution_connect_by_ip', value='Yes', organization=module_manifest_org.id
+    ).create()
+
+    rhel8_contenthost.install_katello_ca()
+    rhel8_contenthost.register_contenthost(module_manifest_org.label, rhel8_module_ak.name)
+    assert rhel8_contenthost.subscribed
+    add_remote_execution_ssh_key(rhel8_contenthost.ip_addr)
+    host = rhel8_contenthost.nailgun_host
+    host = host.read()
+    # Assert no errata on host, no packages applicable or installable
+    errata = _fetch_available_errata(module_manifest_org, host, expected_amount=0)
+    assert len(errata) == 0
+    _run_remote_command_on_content_host(
+        module_manifest_org, "yum install -y katello-host-tools", rhel8_contenthost
+    )
+    # install older version of module stream to generate the errata
+    _run_remote_command_on_content_host(
+        module_manifest_org,
+        f'yum -y module install {module_name}:{stream}:{version}',
+        rhel8_contenthost,
+    )
```

Comment:
Can they both be replaced by two `result = rhel8_contenthost.execute(cmd)` and assert the result.status??

Reply:
Updates made and had to rebase